models/observationSubmissions.js

```diff
@@ -90,5 +90,11 @@ module.exports = {
     userRoleInformation : Object,
     criteriaLevelReport : Boolean,
     userProfile : Object
-  }
+  },
+  compoundIndex: [
+    {
+      "name" :{ createdBy: 1, observationId: 1, submissionNumber:1 },
```

Comment:
@priyanka-TL i think this should have observationId: 1, submissionNumber:1  and entityId :1

Reply:
when we observe again entityId can be same. For same entity also we can observe again